src/mono/wasm/debugger/BrowserDebugProxy/DebugStore.cs

```diff
@@ -370,7 +370,7 @@ public MethodInfo(AssemblyInfo assembly, string methodName, int methodToken, Typ
             _asyncScopes = Array.Empty<AsyncScopeDebugInformation>();
         }
 
-        public MethodInfo(AssemblyInfo assembly, MethodDefinitionHandle methodDefHandle, int token, SourceFile source, TypeInfo type, MetadataReader asmMetadataReader, MetadataReader pdbMetadataReader)
+        public MethodInfo(AssemblyInfo assembly, MethodDefinitionHandle methodDefHandle, int token, SourceFile source, TypeInfo type, MetadataReader asmMetadataReader, MetadataReader pdbMetadataReader, bool fromEnC = false)
```

Comment:
nit: remove the default value for `fromEnC`. This would mean that any code calling this constructor would have to decide whether it is enc or not, which should be useful for new code.